playgrounds/tanstack/start/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "@playgrounds/tanstack-start",
+  "version": "2.0.0",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "dev": "vite dev",
+    "build": "vite build",
+    "start": "node dist/server/server.js",
+    "typecheck": "tsc --noEmit",
+    "lint": "eslint . --max-warnings=0 --cache",
+    "format": "prettier . --ignore-path ../../.prettierignore --check --cache",
+    "yamada-cli": "pnpm -C ../../packages/cli dev"
+  },
```

These paths should be one level higher.
```suggestion
    "format": "prettier . --ignore-path ../../../.prettierignore --check --cache",
    "yamada-cli": "pnpm -C ../../../packages/cli dev"
  },
```